parametric/test_trace_distributed.py

```diff
@@ -62,8 +62,7 @@ def test_distributed_headers_extract_datadog_invalid_D002(test_agent, test_libra
 @pytest.mark.skip_library("golang", "not implemented")
 @pytest.mark.skip_library("nodejs", "not implemented")
 def test_distributed_headers_inject_datadog_D003(test_agent, test_library):
-    """Ensure that Datadog distributed tracing headers are injected properly.
-    """
+    """Ensure that Datadog distributed tracing headers are injected properly."""
```

Comment:
can we do the formatting changes separately? It makes this PR harder to review and introduces irrelevant changes

Reply:
yeah, sorry about that! something went wrong with black, but i reverted the changes